lib/submit-analytics.js

```diff
@@ -23,6 +23,76 @@
 
 'use strict';
 
+const { resolve } = require('path');
+let utils = require('./utils');
+
+const nodeRequire = require;
+
+class WebhookProdDetails {
+    /**
+     * Path and credentials required to submit analytics through the webhook (production mode).
+     */
+    constructor() {
+        this.urlPrefix =
+                "https://webhooks.mongodb-realm.com/api/client/v2.0/app/realmsdkmetrics-zmhtm/service/metric_webhook/incoming_webhook/metric?ip=1&data=";
+        this.urlSuffix = "&ip=1";
+    }
+
+    /**
+     * Constructs the full URL that will submit analytics to the webhook.
+     * @param  {Object} payload Information that will be submitted through the webhook.
+     * @returns {string} Complete analytics submission URL
+     */
+    buildRequest(payload) {
+        const request = this.urlPrefix + Buffer.from(JSON.stringify(payload.webHook), 'utf8').toString('base64') + this.urlSuffix;
```

Comment:
I see little need for a suffix since url query parameters can be considered unordered.
```suggestion
        const request = this.urlPrefix + Buffer.from(JSON.stringify(payload.webHook), 'utf8').toString('base64');
```

Reply:
Fixed in #3524 .